components/CredentialDetailsViews.vue

```diff
@@ -167,10 +169,10 @@ export default {
     const slideNumber = ref(1);
     const tab = ref('highlights');
     const fullscreen = ref(false);
-    const showDetails = ref(false);
     const showDisplays = ref(false);
     const showHighlights = ref(false);
     const credentialImages = reactive([]);
+    const showDetails = ref(process.env.NODE_ENV === 'development');
```

Comment:
This code runs in the browser, so we can't use node.js environment variables for this. We probably just need it to say "(Developer Only)" or something for now, so that regular users don't assume it's safe to use.

Reply:
@dlongley 
Curious what you think about allowing this feature to be controlled by a configuration flag?

_Committed in 361c1e7_